code/game/machinery/scav_selling_pad.dm

```diff
@@ -0,0 +1,205 @@
+/obj/machinery/scav_selling
+	name = "cargo hold pad"
+	icon = 'icons/obj/telescience.dmi'
+	icon_state = "lpad-idle-o"
+	var/idle_state = "lpad-idle-o"
+	var/warmup_state = "lpad-idle"
+	var/sending_state = "lpad-beam"
+	var/cargo_hold_id
+
+/obj/machinery/scav_selling/multitool_act(mob/living/user, obj/item/multitool/I)
+	if (istype(I))
```

Comment:
Is it even possible for this check to fail?

Reply:
no clue it there on the pirate one so i put it there on the scav one